db/migrate/20171110172056_devise_create_users.rb

```diff
@@ -6,6 +6,7 @@ def change
       t.string :encrypted_password, null: false, default: ""
       t.string :first_name, null: false
       t.string :last_name, null: false
+      t.string :locale, null: false
```

Comment:
Remember that you shouldn't edit migrations, you should create  a new one that edits the schema